pyproject.toml

```diff
@@ -110,9 +110,12 @@ SQLAlchemy = ">=1.4.0,<1.5.0"
 sklearn-crfsuite = "~0.3"
 psycopg2-binary = ">=2.8.2,<2.10.0"
 python-dateutil = "~2.8"
-tensorflow = "~2.7.0"
+tensorflow = { version = "~2.8.2", markers="platform_machine == 'x86_64' or platform_machine == 'AMD64'"}
```

Comment:
could we use the inverse of the condition on `tensorflow-macos` to avoid a situation where neither of them is required?

i haven't checked, but does `sys_platform != 'darwin' or platform_machine != 'arm64'`

Reply:
yeah that should work